src/routes/languages/+page.svelte

```diff
@@ -7,26 +7,66 @@
 
 <Stripe>The Languages of Agma Schwa</Stripe>
 <section>
-    <h3 class="text-left">Naturalistic Conlangs</h3>
+    <h3 class="text-left">Conlangs from the world of Djogawa Djororx</h3>
+    
+    <p>These are the languages spoken by the dog people ("pups") in the canon of the <i>In Pursuit of Bez</i> universe, 
```

Comment:
```suggestion
    <p>These are the languages spoken by the dog people (“pups”) in the canon of the <em>In Pursuit of Bez</em> universe, 
```
These should be typographic quotes

Reply:
Also, unless there’s a reason not to, prefer `<em>` over `<i>` everywhere.